src/components/AbstractDisplay/index.js

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react';
+import ReactHtmlParser from 'react-html-parser';
+
+import { Typography, Button, withStyles } from '@material-ui/core';
+
+import './styles.css';
+
+const StyledButton = withStyles({
+    root: {
+        padding: '5px',
+    },
+    label: {
+      textTransform: 'none',
+      textAlign: 'left',
+    },
+  })(Button);
+
+export const AbstractDisplay = ({ abstract, highlight }) => {
+    const [isFullTextToggled, setFullTextToggled] = useState(false);
+    
+    const previewAbstract = highlight && highlight.abstract
+     ? `...${highlight.abstract[0]}...` : abstract;
+
+    const displayFullAbstract = highlight && highlight.abstract
+     ? highlight.abstract.join(' ') : abstract;
```

Comment:
We need to show abstract text instead of all highlights from elastic

Reply:
OK that was my original idea. The reason I changed it was to potentially have the bold text on the expanded article. Is this relevant?